src/sections/Projects/Sistent/identity/color/code.js

```diff
@@ -26,6 +25,1029 @@ import { useStyledDarkMode } from "../../../../../theme/app/useStyledDarkMode";
 const ColorCode = () => {
   const { isDark } = useStyledDarkMode();
   const location = useLocation();
+  const getColorColumns = (isPlusRole) => {
+    if (isPlusRole) {
+      return ["Token", "Value", "Description"];
+    }
+    return ["Token", "Value"];
+  };
+  const colorData = [
```

Comment:
make this const data in a separate file an import it or , do some thing , should not declare constants here as it is very large

Reply:
resolved